src/nodes.js

```diff
@@ -1,4 +1,5 @@
 const { align, breakParent, concat, dedent, dedentToRoot, group, hardline, ifBreak, indent, join, line, lineSuffix, literalline, markAsRoot, softline, trim } = require("prettier").doc.builders;
+const { removeLines } = require("prettier").doc.utils;
```

Comment:
Just wanting to avoid some merge conflicts for now; If the inflight pull requests get merged/closed, I can run prettier over this file and modify this import to be:

```js
const {
  builders: {
    align,
    etc,
    trim
  },
  utils: {
    removeLines
  }
} = require("prettier").doc;
```